hypothesis-python/tests/cover/test_functions.py

```diff
@@ -75,7 +75,7 @@ def test_functions_lambda_with_arg(f):
     assert isinstance(f(1), bool)
 
 
```

Comment:
The second parametrize arg set, won't raise an `InvalidArgument` anymore, since `return = None` gets converted to `st.none()`

Reply:
Let's keep that arg set, but change it to `(lambda: None, "not a strategy")`.

Reply:
Sure 👍